app/page.tsx

```diff
@@ -4,13 +4,16 @@ import React from 'react'
 import CreateSplit from '../src/components/CreateSplit'
 import '../style/global.css'
 import Providers from '../providers/Providers'
+import { base, baseSepolia } from 'viem/chains'
+
+const chain = process.env.NEXT_PUBLIC_TESTNET ? baseSepolia : base
```

Comment:
can you move this to the `const` lib?
- `export const CHAIN = process.env.NEXT_PUBLIC_TESTNET ? baseSepolia : base`